pycti/connector/opencti_connector_helper.py

```diff
@@ -1130,8 +1161,38 @@ def send_stix2_bundle(self, bundle: str, **kwargs) -> list:
             )
             return []
 
-        if entities_types is None:
-            entities_types = []
+        # If directory setup, write the bundle to the target directory
+        if bundle_send_to_directory:
+            self.connector_logger.info(
+                self.connect_name + " sending bundle to directory"
+            )
```

Comment:
Would it be interesting to show the directory in the logger as well ?

` self.connector_logger.info(
                "The connector sending bundle to directory",
                {"connector": self.connect_name, "directory": bundle_send_to_directory}
            )`

Reply:
Done